### PR TITLE
buffer sample

### PR DESCRIPTION
**Pull request by tschie**
shows Geometry Engine's buffer method

### CODE REVIEW

File: src/main/java/com/esri/samples/geometry/buffer/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+<p><img src="Buffer.png"></p>
+
+<h2>How to use the sample</h2>
+<p>Tap on the map to create a buffer around the tapped location. Change the spinner value (in miles) to set the 
+buffer distance.</p>
+
+<h2>How it works</h2>
+<ol>
+<li>The static method, <code>GeometryEngine.buffer</code>, is called by passing in a <code>Point</code> and a distance
```
Comment (john0005) [inline src/main/java/com/esri/samples/geometry/buffer/README.md:18]: Would say, and a distance in meters. This returns a <code>Polygon</code> which can be used to create a <code>Graphic</code> to display it.

File: src/main/java/com/esri/samples/geometry/buffer/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+<h2>Features</h2>
+<ul>
+<li>GeometryEngine.Buffer</li>
+<li>GraphicsOverlay</li>
+<li>Point</li>
+<li>Polygon</li>
+</ul>
+
+<h2>Tags</h2>
```
Comment (john0005) [inline src/main/java/com/esri/samples/geometry/buffer/README.md:30]: Guessing that this is something new we are adding.

File: src/main/java/com/esri/samples/geometry/buffer/BufferSample.java [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+      ArcGISMap map = new ArcGISMap(SpatialReferences.getWebMercator());
+      map.setBasemap(Basemap.createTopographic());
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      Geometry startingEnvelope = new Envelope(-10863035.97, 3838021.34, -10744801.344, 3887145.299,
+          SpatialReferences.getWebMercator());
+      mapView.setViewpointGeometryAsync(startingEnvelope);
+
+      // create a graphics overlay to contain the geometry to clip
```
Comment (john0005) [inline src/main/java/com/esri/samples/geometry/buffer/BufferSample.java]: Instead of `the geometry to clip` would say  buffered geometry.

**Review by john0005 — APPROVED** [review]
Slight changes but looks good!

File: src/main/java/com/esri/samples/geometry/buffer/README.md (hunk 2 above)
Comment (tschie) [reply src/main/java/com/esri/samples/geometry/buffer/README.md:30]: @john0005 Yes, there's a new common readme spec: https://devtopia.esri.com/runtime/common-samples/wiki/Standard-sample-documentation-template-%28ReadMe.md%29. However, the sample viewer doesn't support this yet. I'll hopefully get to that this sprint or the next.

**Pull request merged**